services/QuillLMS/engines/evidence/lib/evidence/evidence/synthetic/seed_data_generator.rb

```diff
@@ -150,13 +150,18 @@ def run
 
       private def parse_completion_api_results(api_results, current_texts:, seed:, noun: nil, label: nil)
         api_results
+          .map {|s| lowercaser.run(s) }
           .map {|s| Result.new(text: [noun, s].join(SPACE).strip, seed: seed, label: label)}
           .uniq {|r| r.text }
           .reject {|r| r.text.in?(current_texts)}
           .reject {|r| regex_exclude?(r.text) }
           .reject {|r| opinion_api_flagged?(r.text) }
       end
 
+      private def lowercaser
```

Comment:
Note, not using `ApplicationService` pattern, e.g. `Evidence::SafeFirstLowercaser.new(passage, text)`, since I don't want to parse the passage each time.